### PR TITLE
zh: update migrate TiKV Pod

### LINKED ISSUE #1400: Add procedure to reschedule a TiKV Pod using EBS to another node
## Change Request

**This repository is ONLY used to solve issues related to TiDB Operator documentation.
For other issues (related to TiDB, PD, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.

To do that gracefully:
Add evict leader scheduler for the store, cordon the node, delete the tikv pod, after the Pod is ready, delete the evict leader scheduler for the store.
2. Describe your suggestion or addition.

3. Provide some reference materials (documents, websites, etc) if you could.

### PR DESCRIPTION
**Pull request by better0332**
### What is changed, added, or deleted? (Required)

update migrate TiKV Pod

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): 

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

Fix #1400

### CODE REVIEW

**Review by july2993 — COMMENTED** [review]
do we need to clarify the supporting version of operator?

**Comment by DanielZhangQD** [comment]
> do we need to clarify the supporting version of operator?

@better0332 @july2993 This change only applies to the master branch, so it will only be in >=v1.3 docs.

File: zh/maintain-a-kubernetes-node.md [hunk 1]
```diff
@@ -311,32 +311,22 @@ TiDB 是高可用数据库，可以在部分数据库节点下线的情况下正
     kubectl get pod --all-namespaces -o wide | grep ${node_name} | grep tikv
     ```
 
-3. 参考[迁移 TiKV Region Leader](#迁移-tikv-region-leader) 将 Leader 迁移到其他 Pod。
-
-4. 删除 TiKV Pod：
+3. 为 TiKV Pod 添加一个 key 为 `tidb.pingcap.com/evict-leader` 的 annotation，触发优雅重启：
```
Comment (DanielZhangQD) [inline zh/maintain-a-kubernetes-node.md:314]:
```suggestion
3. 为 TiKV Pod 添加一个 key 为 `tidb.pingcap.com/evict-leader` 的 annotation，触发优雅重启，TiDB Operator 会在迁移完 TiKV region leader 后删掉 Pod：
```

File: zh/maintain-a-kubernetes-node.md [hunk 2]
```diff
@@ -311,32 +311,22 @@ TiDB 是高可用数据库，可以在部分数据库节点下线的情况下正
...
     ```
 
-5. 确认该 TiKV Pod 正常调度到其它节点上：
+4. 确认该 TiKV Pod 正常调度到其它节点上：
 
     {{< copyable "shell-regular" >}}
 
     ```shell
     watch kubectl -n ${namespace} get pod -o wide
     ```
```
Comment (DanielZhangQD) [inline zh/maintain-a-kubernetes-node.md:329]: 参考 L454 加一步检查 leader 开始调度回来

File: zh/maintain-a-kubernetes-node.md [hunk 3]
```diff
@@ -311,30 +311,28 @@ TiDB 是高可用数据库，可以在部分数据库节点下线的情况下正
...
-4. 删除 TiKV Pod：
+3. 为 TiKV Pod 添加一个 key 为 `tidb.pingcap.com/evict-leader` 的 annotation，触发优雅重启，TiDB Operator 会在迁移完 TiKV region leader 后删掉 Pod：
 
     {{< copyable "shell-regular" >}}
-
-    ```shell
-    kubectl delete -n ${namespace} pod ${pod_name}
+   
+    ```bash
+    kubectl -n ${namespace} annotate pod ${tikv_pod_name} tidb.pingcap.com/evict-leader="delete-pod"
```
Comment (DanielZhangQD) [inline zh/maintain-a-kubernetes-node.md:319]:
```suggestion
    kubectl -n ${namespace} annotate pod ${pod_name} tidb.pingcap.com/evict-leader="delete-pod"
```

File: zh/maintain-a-kubernetes-node.md [hunk 4]
```diff
@@ -453,26 +451,18 @@ TiDB 是高可用数据库，可以在部分数据库节点下线的情况下正
...
-1. 查看 TiKV Pod 的 `store-id`：
+1. 为 TiKV Pod 添加一个 key 为 `tidb.pingcap.com/evict-leader` 的 annotation：
 
     {{< copyable "shell-regular" >}}
-
-    ```shell
-    kubectl get tc ${cluster_name} -ojson | jq ".status.tikv.stores | .[] | select ( .podName == \"${pod_name}\" ) | .id"
+   
+    ```bash
+    kubectl -n ${namespace} annotate pod ${tikv_pod_name} tidb.pingcap.com/evict-leader="none"
```
Comment (DanielZhangQD) [inline zh/maintain-a-kubernetes-node.md:459]:
```suggestion
    kubectl -n ${namespace} annotate pod ${pod_name} tidb.pingcap.com/evict-leader="none"
```

File: zh/maintain-a-kubernetes-node.md [hunk 5]
```diff
@@ -311,30 +311,28 @@ TiDB 是高可用数据库，可以在部分数据库节点下线的情况下正
...
-6. 移除 evict-leader-scheduler，等待 Region Leader 自动调度回来：
+5. 检查 Region Leader 已经开始迁回:
 
-    {{< copyable "shell-regular" >}}
+   {{< copyable "shell-regular" >}}
 
-    ```shell
-    pd-ctl scheduler remove evict-leader-scheduler-${ID}
+    ```bash
+    kubectl get tc ${cluster_name} -ojson | jq ".status.tikv.stores | .[] | select ( .podName == \"${pod_name}\" ) | .leaderCount"
```
Comment (DanielZhangQD) [inline zh/maintain-a-kubernetes-node.md:335]:
```suggestion
    kubectl -n ${namespace} get tc ${cluster_name} -ojson | jq ".status.tikv.stores | .[] | select ( .podName == \"${pod_name}\" ) | .leaderCount"
```

File: zh/maintain-a-kubernetes-node.md [hunk 6]
```diff
@@ -453,26 +451,18 @@ TiDB 是高可用数据库，可以在部分数据库节点下线的情况下正
...
-    pd-ctl scheduler add evict-leader-scheduler ${ID}
-    ```
-
-3. 检查 Region Leader 已经全部被迁移走:
-
-    {{< copyable "shell-regular" >}}
-
-    ```shell
+    ```bash
     kubectl get tc ${cluster_name} -ojson | jq ".status.tikv.stores | .[] | select ( .podName == \"${pod_name}\" ) | .leaderCount"
```
Comment (DanielZhangQD) [inline zh/maintain-a-kubernetes-node.md:467]:
```suggestion
    kubectl -n ${namespace} get tc ${cluster_name} -ojson | jq ".status.tikv.stores | .[] | select ( .podName == \"${pod_name}\" ) | .leaderCount"
```

**Review by DanielZhangQD — APPROVED** [review]
LGTM

File: zh/maintain-a-kubernetes-node.md [hunk 7]
```diff
@@ -311,30 +311,28 @@ TiDB 是高可用数据库，可以在部分数据库节点下线的情况下正
     kubectl get pod --all-namespaces -o wide | grep ${node_name} | grep tikv
     ```
 
-3. 参考[迁移 TiKV Region Leader](#迁移-tikv-region-leader) 将 Leader 迁移到其他 Pod。
-
-4. 删除 TiKV Pod：
+3. 为 TiKV Pod 添加一个 key 为 `tidb.pingcap.com/evict-leader` 的 annotation，触发优雅重启，TiDB Operator 会在迁移完 TiKV region leader 后删掉 Pod：
```
Comment (ran-huang) [inline zh/maintain-a-kubernetes-node.md:314]:
```suggestion
3. 为 TiKV Pod 添加一个 key 为 `tidb.pingcap.com/evict-leader` 的 annotation，触发优雅重启，TiDB Operator 会在迁移完 TiKV Region Leader 后删掉 Pod：
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**